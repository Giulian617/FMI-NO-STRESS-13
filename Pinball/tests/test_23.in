100 22142979520776
8 30 3
21 30 8
27 30 10
85 21 3
87 30 9
66 87 5
100 8 9
23 66 10
41 27 8
33 85 7
37 100 3
18 37 4
63 33 5
81 30 2
54 81 6
96 27 5
97 37 8
90 27 2
93 41 8
95 37 10
2 54 1
31 33 3
88 30 7
6 100 7
1 95 9
45 63 10
79 2 5
34 2 9
56 100 3
4 79 1
39 63 9
9 8 5
57 1 7
12 18 4
20 31 10
62 1 9
22 9 2
43 27 1
35 62 8
10 97 10
29 8 2
74 100 4
94 22 6
55 27 2
86 66 10
52 37 10
72 10 3
51 88 5
84 79 8
17 27 7
36 9 10
11 35 1
60 37 4
15 55 7
69 22 6
26 37 1
7 11 8
99 15 7
50 72 5
19 56 8
78 62 5
25 95 1
82 41 8
59 22 7
67 6 10
73 93 1
24 88 6
77 56 3
61 67 10
75 96 7
65 82 1
89 86 1
68 10 8
48 11 1
92 55 7
28 92 6
46 21 10
3 9 3
13 87 4
91 28 6
47 92 2
49 57 8
80 97 2
64 10 5
76 69 10
71 75 2
16 99 8
53 30 7
40 53 9
14 75 7
42 90 10
5 71 3
58 73 1
98 17 4
44 71 2
83 74 1
32 44 8
38 16 8
70 12 9
