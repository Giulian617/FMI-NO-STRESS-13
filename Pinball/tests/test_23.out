888522756
693607855
466419187
998985113
398767546
791577275
445863857
866796474
653561182
353019931
74859306
811323955
879479615
345384188
107200287
514751035
143733510
416544635
821404055
16260291
346061937
454589175
400317944
169265784
323031970
243332235
627008516
118229039
127838897
930737240
580539293
83153986
234159813
838891787
930737240
313486052
83153986
181834063
930483058
116025631
240063191
257983728
448523547
653561182
32133648
181834063
464961090
